shardingsphere-governance/shardingsphere-governance-core/src/main/java/org/apache/shardingsphere/governance/core/registry/RegistryCenter.java

```diff
@@ -480,6 +482,19 @@ public synchronized void renew(final GrantStatementEvent event) {
         persistChangedPrivilege(event.getUsers());
     }
     
+    /**
+     * Load children values.
+     *
+     * @param event get children request event.
+     */
+    @Subscribe
+    public void loadChildrenValues(final GetChildrenRequestEvent event) {
```

Comment:
Why is it called `loadChildrenValues`? I suppose its named relevant with `show process`, like `loadShowProcessData`, `loadShowProcessList`? And the event name is the same case.
Actually, ShardingProxy does not know the detail of `registryCenter`, e.g., child node.  What Proxy wants to do is to request its expected data.